src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,72 @@
  */
 
 public class Plane {
+    private String model;
+    private int countOfSeats;
+    private double price;
+    private int buildYear;
+    private String color;
 
+    public Plane(PlaneBuilder builder) {
+    }
```

Comment:
bring back your previous implementation of this constructor

why have you decided to change it?

Reply:
I didn't understand your first comment, now it should be ok